be/src/vec/exec/format/parquet/vparquet_reader.cpp

```diff
@@ -156,12 +156,13 @@ Status ParquetReader::_init_read_columns() {
             _missing_cols.push_back(file_col_name);
         }
     }
+    // It is legal to get empty include_column_ids in query task.
+    _read_columns.clear();
```

Comment:
It seems we don't need to call this `clear` for `_read_columns`, because `_read_columns` is only initialized once